### ISSUE TITLE
[ELECTRA/TF2] Why is `hvd.DistributedGradientTape` disabled?

### ISSUE THREAD

**Issue by WissamAntoun** [NVIDIA/DeepLearningExamples#1169]
Was the line calling `hvd.DistributedGradientTape()` commented out by mistake? https://github.com/NVIDIA/DeepLearningExamples/blob/master/TensorFlow2/LanguageModeling/ELECTRA/run_pretraining.py#L207 

since it is enabled in the BERT/TF2 trainer.

**Comment by WissamAntoun** [comment]
Apparently, it is because the allreduce operation is being done manually after gradient accumulation. So no need for the hvd gradient tape
